src/main/java/pages/accounts/AccountsPage.java

```diff
@@ -0,0 +1,74 @@
+package pages.accounts;
+
+import base.BasePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+
+public class AccountsPage extends BasePage {
+
+    @FindBy(xpath = "//div[@title = 'New']")
+    private WebElement newAccountButton;
+
+    @FindBy(xpath = "//div//a/div[@title = 'Import']")
+    private WebElement importButton;
+
+    @FindBy(xpath = "//h1[@class = 'slds-truncate slds-page-header__title slds-text-color--default']")
+    private WebElement listViewDropDown;
+
+    @FindBy(xpath = "//ul[@id='virtualAutocompleteListbox_e4947a128442e']/li[2]")
+    private WebElement allAccountsListView;
+
+    @FindBy (xpath = "//input[@id = 'input-101']")
+    private WebElement localSearch;
+
+    @FindBy(xpath = "//a[text() = 'test Account']")
+    private WebElement testAccount;
+
+    @FindBy(xpath = "//div[@title = 'Edit']")
+    private WebElement editButtonOnTestAccount;
+
+    @FindBy(xpath = "")
+    private List<WebElement> accountNames;
+
+    public CreateNewAccountsPage clickOnNewAccountButton () throws InterruptedException {
```

Comment:
Change method's name to more understandable. If this button navigates to New Account creation form, then it can be called like 'createAccount'